tutor/commands/dev.py

```diff
@@ -120,6 +120,26 @@ def quickstart(
     )
 
 
+@click.command(help="Configure and run Open edX from scratch, for development")
+@click.option("-I", "--non-interactive", is_flag=True, help="Run non-interactively")
+@click.option("-p", "--pullimages", is_flag=True, help="Update docker images")
+@compose.mount_option
+@click.pass_context
+def quickstart(
+    context: click.Context,
+    non_interactive: bool,
+    pullimages: bool,
+    mounts: t.Tuple[t.List[compose.MountParam.MountType]],
+) -> None:
+    """
+    This command has been renamed to 'launch'.
+    """
+    fmt.echo_alert(
+        "The 'quickstart' command is deprecated and will be removed in a later release. Use 'launch' instead."
+    )
+    launch(context, non_interactive, pullimages, mounts)
```

Comment:
This is not the right way to call a different click command. Instead, we should write:

    context.invoke(quickstart, non_interactive=non_interactive, ...)

See click docs: https://click.palletsprojects.com/en/7.x/api/#click.Context.invoke

Same comment applies to other quickstart commands below.

Reply:
I've changed this in dev.py, local.py, and k8s.py